test/e2e/tests/book-appointment.spec.ts

```diff
@@ -14,11 +14,24 @@ const verifyBookingPageLoaded = async () => {
   await expect(bookingPage.invitingText).toBeVisible();
   await expect(bookingPage.invitingText).toContainText(PROD_DISPLAY_NAME);
   await expect(bookingPage.bookingCalendar).toBeVisible();
-  // calendar header should contain current MMM YYYY
+
+  // verify calendar header
   const today: Date = new Date();
   const curMonth: string = today.toLocaleString('default', { month: 'short' });
   const curYear: string = String(today.getFullYear());
-  await expect(bookingPage.calendarHeader).toHaveText(`${curMonth} ${curYear}`);
+
+  // expect correct year
+  await expect(bookingPage.calendarHeader).toContainText(curYear);
```

Comment:
This wouldn't work at the end of the year, when the switch is from December to January, or would it?

Reply:
Great, thanks!